risc0/build/src/config.rs

```diff
@@ -24,6 +24,9 @@ pub struct DockerOptions {
     ///
     /// The current working directory is used if `None` is specified.
     pub root_dir: Option<PathBuf>,
+
+    /// Specify the environemnt variables for the docker build.
+    pub compile_time_env: Option<Vec<(String, String)>>,
```

Comment:
Hmmm, this is actually a breaking change. And we do use the `DockerOptions { _ }` syntax to construct this type in downstream applications. I do wish we'd marked this as `#[non_exhaustive]` and gave a builder instead like we do for `ProverOpts` [1]. Maybe we accept this as a breaking change, but it'd be good is there is another way.

[1] https://github.com/risc0/risc0/blob/533aff23f633c6102c961d47b9370ec96770de6d/risc0/zkvm/src/host/client/prove/mod.rs#L159